docs/contributing.rst

```diff
@@ -0,0 +1,267 @@
+.. highlight:: shell
```

Comment:
```suggestion
:orphan:

.. highlight:: shell
```

One way to get rid of the warning

Reply:
this is already sovled by adding the new files into a the strucure and into the hidden toctrees in main.